manifests/init.pp

```diff
@@ -28,7 +29,8 @@
   }
 
   class { "rpmbuilder::packages::essential":
-    epel  => $add_epel,
+    epel         => $add_epel,
+    mock_version => $mock_version,
```

Comment:
Yes, if mock_version is undef, it won't override the default (at least in puppet 3). I think it's worth leaving as undef, rather than encoding the same default everywhere.